tt_metal/hw/ckernels/wormhole_b0/metal/llk_api/llk_sfpu/ckernel_sfpu_typecast.h

```diff
@@ -93,5 +93,42 @@ inline void calculate_typecast_int32_to_fp16b()
     }
 }
 
+template <bool APPROXIMATION_MODE, int ITERATIONS>
+inline void calculate_typecast_fp16b_to_int32()
+{
+    #pragma GCC unroll 0
+    for (int d = 0; d < ITERATIONS; d++) {
+        vFloat in = dst_reg[0];
+
+        // extract exponent
+        vInt exp = exexp(in);
+
+        v_if (exp < 0) {
+            dst_reg[0] = 0;
+        } v_elseif (exp > 30) {
+            // set to int32 max value in case of overflow
+            vInt tmp = 2147483647;
```

Comment:
Can you either add a macro define for this value, or use the ones in gcc compiler (might be called `INT_MAX`?)